skit.Application/Addresses/Queries/DTO/AddressDto.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace skit.Application.Addresses.Queries.DTO;
+
+public sealed class AddressDto
+{
+    public Guid Id { get; set; }
+    public string? City { get; set; }
+    public string? Street { get; set; }
+    public string? HouseNumber { get; set; }
+    public string? PostalCode { get; set; }
```

Comment:
Wszystko tutaj powinno byc nullable?

Reply:
Ja w DTO zawsze daję wszystko nullable. Potem jak się pomyślisz to będzie ci wywalać 500, a tak masz spokój a na nic innego to nie wpływa

Reply:
Wsm to nie jest glupie, tak czy siak na bazie powinno sie zapisac prawidlowo wiec nie ma sensu przy get'ach takiego czegos sprawdzac